### PR TITLE
added total_sample column to the aggs tables and views for desktop and glean

### PR DESCRIPTION
**Pull request by alekhyamoz**
This is related to #1240. 
Added the `total_sample` column to the existing aggregation tables (desktop, fenix, and fog) and their respective materialized views. 

Testing data:
Uploaded test files to project : `glam-fenix-dev ` and bucket : `glam-fenix-dev-testing`

Commands to run: 
`./manage.py migrate `
`./manage.py import_desktop_aggs nightly --bucket glam-fenix-dev-testing`
`./manage.py import_glean_aggs nightly --bucket glam-fenix-dev-testing`

### CODE REVIEW

File: glam/api/migrations/0022_auto_20220303_1530.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0021_fog_counts'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FOGAggregationView',
```
Comment (Iinh) [inline glam/api/migrations/0022_auto_20220303_1530.py]: @alekhyamoz why do we need this model?

Comment (alekhyamoz) [reply glam/api/migrations/0022_auto_20220303_1530.py]: Not required. was included as part of testing. removed it now

**Review by edugfilho — APPROVED** [review]

File: glam/api/migrations/0022_sample_counts_tables.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+sql = []
+for channel in constants.CHANNEL_NAMES.values():
+    sql.extend(
+        [
+            f"DROP MATERIALIZED VIEW view_glam_desktop_{channel}_aggregation;",
+            f"CREATE MATERIALIZED VIEW view_glam_desktop_{channel}_aggregation AS SELECT * FROM glam_desktop_{channel}_aggregation;",
+            f"CREATE UNIQUE INDEX ON view_glam_desktop_{channel}_aggregation (id);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation (version);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation USING HASH (metric);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation USING HASH (os);",
```
Comment (Iinh) [inline glam/api/migrations/0022_sample_counts_tables.py:16]: I hope this won't extend our ETL time by a lot. IIRC this step usually takes the longest right?

**Review by Iinh — APPROVED** [review]
LGTM! Thanks @alekhyamoz!

File: glam/api/migrations/0022_sample_counts_tables.py (hunk 2 above)
Comment (alekhyamoz) [reply glam/api/migrations/0022_sample_counts_tables.py:16]: Yes Linh. That's what is concerning about this release.

**Pull request merged**